tools/build/binary-release/assets/MacOS/README.md

```diff
@@ -27,23 +37,19 @@ following to `~/.bash_profile` or your equivalent:
 
     eval "$(/path/to/this/folder/scripts/set-env.sh --quiet)"
 
-To start an interactive Raku environment call `raku` without an argument
-
-    raku
-
 
-Installing modules
+Installing Modules
 ==================
 
 To install Raku modules you can use the Zef module installer.
 
-    raku /path/to/this/folder/share/perl6/site/bin/zef install JSON::Fast
+    zef install JSON::Fast
 
 Modules will be installed into this Raku package and will thus be available
 even when moving this package.
 
 
-Native code modules
+Native Code Modules
 -------------------
 
 To install modules that require a compiler toolchain, you have to have a
```

Comment:
Maybe clarify here that you're talking about a compiler toolchain *other* than Rakudo itself?